experimental/status/status_source.go

```diff
@@ -136,7 +137,8 @@ func IsDownstreamError(err error) bool {
 func IsDownstreamHTTPError(err error) bool {
 	return IsDownstreamError(err) ||
 		isConnectionResetOrRefusedError(err) ||
-		isDNSNotFoundError(err)
+		isDNSNotFoundError(err) ||
+		isSocksProxyError(err)
```

Comment:
It would be nice if we could avoid doing string comparison for the errors and instead are able to rely on an error type. Depending on the error your experiencing, we might do some working the the `proxy` package to wrap proxy errors so we could actually use `errors.Is` or `errors.As` here.

Context: https://github.com/grafana/grafana-plugin-sdk-go/blob/main/backend/proxy/secure_socks_proxy.go#L302

Reply:
That's also an idea 👍